docs/whatsnew/1.1.rst

```diff
@@ -0,0 +1,182 @@
+        subgraph cluster_astropy {
+                color=blue
+                fontcolor=blue
+                penwidth=2
+                label=<<b>Frames implemented in Astropy</b>>
+                ICRS
+                HCRS
+                HeliocentricMeanEcliptic
+                astropy [label="Other Astropy frames" shape=box3d style=filled]
+                geocentric [label="Earth-centered frames (including GEO)" shape=box3d style=filled]
+                astropy -> ICRS
+                geocentric -> ICRS
+                ICRS -> astropy
+                ICRS -> geocentric
+        }
+        subgraph cluster_sunpy {
+                color=crimson
+                fontcolor=crimson
+                penwidth=2
+                label=<<b>Frames implemented in SunPy</b>>
+                Helioprojective
+                Heliocentric
+                HeliographicStonyhurst
+                HeliographicCarrington
+                subgraph cluster_sunpy11 {
+                        color=chocolate
+                        fontcolor=chocolate
+                        label=<<b>Requires SunPy 1.1</b>>
+                        HeliocentricInertial
+                        HeliocentricEarthEcliptic
+                        GeocentricSolarEcliptic
+                        GeocentricEarthEquatorial
+                }
+        }
+        newrank=true
+   }
+
+.. _whatsnew-1.1-dl_manager:
+
+Manager for Versioned Data Files
+================================
+
+SunPy 1.1 provides a data manager for versioning and caching remote files. Given the URL of a remote file and a SHA256 hash, the `~sunpy.data.manager.require` decorator can be added to functions that require specific data files from remote sources (e.g. computing instrument response functions). If the specified hash does not match that of the remote version, an exception is raised to make the user aware of any changes on the remote server or tampering with local files. Additionally, `~sunpy.data.cache` can be used to avoid re-downloading files that already exist on a user's local machine, thus saving disk space and internet bandwidth.
+
+
+.. _whatsnew-1.1-SUVI:
+
+Support for SUVI Data
+=====================
+
+The Solar Ultraviolet Imager (SUVI) is a EUV instrument imaging the full disk of the Sun in six passbands, and is onboard the latest of the Geostationary Operational Environmental Satellite (GOES) missions.  `~sunpy.map.SUVIMap` provides SunPy map support for SUVI FITS image data.
+
+.. _whatsnew-1.1-WISPR:
+
+Initial Support for WISPR Images
+================================
+
+Following the first data release from Parker Solar Probe, SunPy 1.1 supports loading WISPR imaging data into a `~sunpy.map.GenericMap`.
+Due to the complex projections in the WISPR data this involved changing the way sunpy converts FITS headers into `astropy.wcs.WCS` objects.
+It is expected that sunpy 2.0 will include more complete support for WISPR and other PSP data.
```

Comment:
Which other PSP data? WISPR is the only imaging instrument.

Reply:
Is there timeseries data we should be supporting?

Reply:
Ah good point, there's radio data, and SEP data that SunPy probably wants to support so this is fine as is.